topostats/theme.py

```diff
@@ -78,3 +82,21 @@ def nanoscope():
 
         colormap = LinearSegmentedColormap.from_list("nanoscope", vals, N=256)
         return colormap
+
+    @staticmethod
+    def gwyddion():
```

Comment:
I'm naive, but how does this differ qualitatively from `nanoscope`?

#367 documents that the colourmaps are slightly off, if this corrects that why is a new colourmap being introduced rather than correcting `nanoscope`?

Reply:
No, worries, thanks for asking :) 

Here I was just testing if the colourmaps could easily be created from the values inside the Gwyddion software. Gywddion.net (or `gwyddion` here) is one such widely used and industry standard colormap that when I saw it work I thought we should just keep it in incase anyone has a preference for it. 

nanoscope is the just our labs preferred publication colourmap but there are others, such as this one.

This led me to add to #367 with the steps for someone to replicate the the colourmap without simulating an spm image which I'm unsure how easy that is. If you want I could try and obtain those values and fix that issue in this PR?

Reply:
I think keeping PRs small and modularising issues is best, so probably keep it to its own issue and PR? Not an expert here though.